frontend/packages/kubevirt-plugin/src/selectors/vm-template/selectors.ts

```diff
@@ -55,6 +58,9 @@ export const getTemplatesLabelValues = (templates: TemplateKind[], label: string
   return labelValues;
 };
 
+export const getTemplateValidationsFromTemplate = (vmTemplate: TemplateKind): TemplateValidations =>
+  new TemplateValidations(iGetTemplateValidations(fromJS(vmTemplate)));
```

Comment:
this should be a basic selector. It is very inefficient to create an immutable to just select some stuff

Reply:
done